packages/features/bookings/Booker/Booker.tsx

```diff
@@ -173,13 +174,11 @@ const BookerComponent = ({
             <StickyOnDesktop
               key="meta"
               className={classNames(
-                "relative z-10 flex [grid-area:meta]",
+                "relative z-10 flex flex-col [grid-area:meta]",
                 // Important: In Embed if we make min-height:100vh, it will cause the height to continuously keep on increasing
                 layout !== BookerLayouts.MONTH_VIEW && !isEmbed && "sm:min-h-screen"
               )}>
-              <BookerSection
-                area="meta"
-                className="max-w-screen flex w-full flex-col md:w-[var(--booker-meta-width)]">
```

Comment:
this looks dangerous. is this needed?

Reply:
@PeerRich I have adjusted it because area has no effect on the display. From my point of view this is a remnant and its not needed.